tests/log-in.test.js

```diff
@@ -0,0 +1,75 @@
+//const { default: $ } = require("webdriverio/build/commands/browser/$");
+describe('Banking', () => {
+
+    it('Debería ingresar a la pagina con datos correctos', async () => {
+        await browser.url('login');
+
+        let userName = await $('#username');
+        await userName.setValue('jsmith@demo.io');
+
+        let pssWord = await $('#password');
+        await pssWord.setValue('Demo123!');
+
+        let submit = await $('#submit');
+        await submit.click();
+
+        let url = await browser.getUrl(); 
+        //await console.log(url);
+        //await console.log("salida");
+
+        await expect(url).to.equal('http://localhost:8080/bank/home');
+
+        //await browser.pause(5000);
+    });
+    
+    it('Debería recordar sesión iniciada', async () => {
```

Comment:
Every other test name you defined has an overall correct structure (ignoring they're in spanish, but I already made a comment about it), but this one hasn't.

Consider using, whenever possible, a naming structure that emphasizes what **should** happen **when** some action is performed / the application is in a given state. This makes it easy to figure out what to expect from the test without seeing its implementation.

Eg. "Should redirect to homepage when accessing login page with a 'remember me' option"